packages/webapp/src/components/Drawer/index.tsx

```diff
@@ -29,63 +47,69 @@ const Drawer = ({
   classes = {
     modal: '',
     drawer: '',
-    backdrop: '',
```

Comment:
I wanted to be explicit (because I was using them assuming something different -- i.e. that I could use the backdrop class to style both the drawer + the modal) that these props only apply to the drawer state; there actually was zero way to style the modal state using this classes object as written.